packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Icon, IIconStyles, Stack } from '@fluentui/react';
+import { NotificationIconStyles, NotificationTextStyles } from './styles/MeetingCompositeStyles';
+
+/**
+ * @private
+ */
+export type NotificationIconProps = {
+  numberOfMessages: number;
+};
+/**
+ * @private
+ */
+export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
+  const { numberOfMessages } = props;
+
+  const renderNumber = (numberOfMessages): JSX.Element => {
+    if (numberOfMessages < 1) {
+      return <></>;
+    } else {
+      return <Stack styles={NotificationTextStyles}>{numberOfMessages < 9 ? <>{numberOfMessages}</> : <>+9</>}</Stack>;
```

Comment:
1. You don't need `<></>` since these are already wrapped in a `Stack`.
2. btw, *why are they in a `Stack`*? A `Stack` only makes sense if you have more than one item. You should probably use a `Text` https://developer.microsoft.com/en-us/fluentui#/controls/web/text
3. Also, double check what aria labels to set so that announcer correctly handles the notification.

[Wrote this comment then realized it was three comments :P]

Reply:
will take a look at this!

Reply:
in reply to 2. they are in a stack so that the styles at the bottom of the page can be applied to them to fit within the icon appropriately.

Reply:
for 3. the aria labels will be added in another PR? I imagine that I will need to add strings to support the reading of other locales?